gulpfile.js

```diff
@@ -52,24 +51,15 @@ gulp.task('compile', gulp.series(
   buildSassdocs
 ))
 
-/**
- * Serve task
- * Restarts Node.js app when there are changes
- * affecting .js, .mjs and .json files
- */
-gulp.task('serve', gulp.parallel(
-  'watch',
-  runNodemon
-))
-
 /**
  * Dev task
  * Runs a sequence of tasks on start
  */
 gulp.task('dev', gulp.series(
   cleanPublic,
   'compile',
-  'serve'
+  'watch',
```

Comment:
Are there any "gotchas" we might miss running these tasks in slightly different order?

Reply:
Not that I could see (as Nodemon does its own watching) and no compile steps are required for Express to start

Reply:
We still have `'compile'` coming first, so browser assets served by Express will already be in place too